### ISSUE TITLE
Party sheet should show non-compendium abilities

### ISSUE THREAD

**Issue by marksjus** [lumphammer/gumshoe-fvtt#835]
Party Actor uses only abilities from PC compendium pack configured in the system and disregards any manual changes.

When you configure the system to not use any default PC abilities (no compendium pack selected) and add abilities to the player manually or using compendium pack attached to a personalDetail Item, the Party Actor list is empty. Also, any custom Ability added only to selected players is also missing in Party Actor list.

**Comment by n3dst4** [comment]
Good point. The original idea, as you probably understand, was that the compendium defines your system. But I guess if you're not using a compendium that makes the party sheet rather unhelpful. I will tag this as a feature rather than a bug though.

**Comment by marksjus** [comment]
Sure, in 99,99% cases nobody will notice that. I found out about it only because I was messing with custom abilities for selected players.
